src/main/java/org/apache/sling/dynamicinclude/Configuration.java

```diff
@@ -66,11 +65,7 @@
       @AttributeDefinition(name="Resource types", description="Filter will replace components with selected resource types", cardinality = Integer.MAX_VALUE)
       String include$_$filter_config_resource$_$types() default "";
       
-      @AttributeDefinition(name = "Include type", description = "Type of generated include tags", options = {
-          @Option(label = "Apache SSI", value = "SSI"),
-          @Option(label = "ESI", value = "ESI"),
-          @Option(label = "Javascript", value = "JSI")
-      })
+      @AttributeDefinition(name = "Include type", description = "Type of generated include tags")
```

Comment:
Any reason for removing this?

Reply:
well, if the point is to allow extra implementations, that means we cannot longer have a fixed list of options. otherwise you cannot choose whatever you implement

Reply:
Makes sense. It might help if we listed the built-in implementations.